src/AppBundle/Repository/EventRepository.php

```diff
@@ -172,7 +185,7 @@ public function getPagesUsingFile(string $dbName, string $filename): int
             $rqb->select('COUNT(DISTINCT(CONCAT(gil_wiki, gil_page)))')
                 ->from('commonswiki_p.globalimagelinks')
                 ->where('gil_to = :filename')
-                ->andWhere('gil_page_namespace_id = 0');
+                ->andWhere('gil_page_namespace_id = 6');
```

Comment:
This query looks at global file usage. We only want to count mainspace pages, right? I think this is what's causing the build to fail.

Reply:
D'oh!